packages/authorization-process/src/services/validators.ts

```diff
@@ -126,3 +127,22 @@ export const assertKeyDoesNotAlreadyExist = async (
     throw keyAlreadyExists(kid);
   }
 };
+
+export function assertValidateRsaKeyLength(
+  base64Key: string,
+  minLength: number = 2048
+): void {
+  const bitLength = getByteLengthFromBase64(base64Key) * 8;
+  // Verifica la lunghezza
+  if (bitLength < minLength) {
+    throw invalidKeyLenght(base64Key);
+  }
+}
+
+function getByteLengthFromBase64(base64Key: string): number {
+  // remove padding '=' from the end of the base64 string
+  const trimmedBase64 = base64Key.replace(/=+$/, "");
+  const byteLength = (trimmedBase64.length * 3) / 4;
+
+  return Math.floor(byteLength);
```

Comment:
```suggestion
  const key: KeyObject = createPublicKey(base64Key);

  if (
    key.asymmetricKeyDetails?.modulusLength &&
    key.asymmetricKeyDetails?.modulusLength < minLength
  ) {
    throw invalidKeyLength(base64Key);
  }
```
You can access its size from the KeyObject by using the createKey that is already defined in commons.

Since in this way we are already creating a key you could refactor the main function in order to use it instead of recreateing the key when needed.